default.nix

```diff
@@ -10,14 +10,14 @@ let
       propagatedBuildInputs = [
         python.pkgs.numpy
       ];
-      meta = with pkgs.stdenv.lib; {
+      meta = with pkgs.lib; {
         homepage = "https://github.com/Muges/audiotsm";
         license = licenses.mit;
         description = "A real-time audio time-scale modification library";
       };
     };
 
-  pythonForThis = python.withPackages (ps: with ps;[
+  pythonForThis = python310.withPackages (ps: with ps;[
```

Comment:
Why specifically lock into python 3.10? Should it not be better to just specify to use version 3 in general?
Furthermore why only on this stage, audiotsm would then still be build for python2.
Should it not be sufficient to instead just change line 4 to `python = python3` instead of doing it down here?